src/components/20-molecules/footer-small/index.js

```diff
@@ -21,29 +28,80 @@ class AXAFooterSmall extends LitElement {
     this.languageItems = [];
     this.disclaimerItems = [];
     this.copyrightText = '';
+    this.activeLanguage = '';
+    this.dynamic = false;
+    this.onLanguageChange = () => {};
+    this.onDisclaimerChange = () => {};
   }
 
-  renderFooterList = items => html`
-    <ul class="m-footer-small__list">
-      ${repeat(
-        items,
-        item => html`
-          <li class="m-footer-small__list-item">
-            <axa-link class="m-footer-small__link" href="${item.link}" color="white">${item.text}</axa-link>
-          </li>
-        `
-      )}
-    </ul>
-  `;
+  handleLanguageClick = (ev, language) => {
+    ev.preventDefault();
+    if (this.activeLanguage !== language) {
+      this.activeLanguage = language;
+      this.onLanguageChange(language);
+      this.dispatchEvent(new CustomEvent('axa-language-change', { detail: language, bubbles: true, cancelable: true }));
+    }
+  };
+
+  handleDisclaimerClick = (ev, disclaimer) => {
+    ev.preventDefault();
+    this.onDisclaimerChange(disclaimer);
+    this.dispatchEvent(new CustomEvent('axa-disclaimer-change', { detail: disclaimer, bubbles: true, cancelable: true }));
+  };
 
   render() {
     return html`
       <article class="m-footer-small">
-        ${this.renderFooterList(this.languageItems)}
+        <ul class="m-footer-small__list">
+          ${repeat(
+            this.languageItems,
+            languageItem => html`
+              <li class="m-footer-small__list-item">
+                ${this.dynamic
+                  ? html`
+                      <axa-link
+                        class="m-footer-small__link ${languageItem.text === this.activeLanguage ? 'm-footer-small__link--is-active' : ''}"
+                        color="white"
+                        @click="${ev => this.handleLanguageClick(ev, languageItem.text)}"
+                        >${languageItem.text}</axa-link
+                      >
+                    `
+                  : html`
+                      <axa-link
```

Comment:
really nuice now! The only questionable thing is the "need" for dynamic a tag links, that could be a span or whatever. That can be discussed with @LucaMele later and shouldn't a merge blocker.